tests/test_all.py

```diff
@@ -101,6 +103,14 @@ def test_glibc_present(auto_container):
         assert auto_container.connection.exists(binary)
 
 
+@pytest.mark.parametrize(
+    "container_per_test", CONTAINERS_WITH_ZYPPER, indirect=True
+)
+def test_zypper_up_works(container_per_test: ContainerData) -> None:
+    """Check that :command:`zypper -n up` works."""
+    container_per_test.connection.run_expect([0], "zypper -n up")
```

Comment:
btw, does this command have a default timeout?

Reply:
I don't think it has one

Reply:
I added a `timeout`